### PR TITLE
Added now and today helpers

### PR DESCRIPTION
**Pull request by nyanlynnhtut**

### CODE REVIEW

**Comment by taylorotwell** [comment]
Subject verb disagreement.

File: helpers.md [hunk 1]
```diff
@@ -980,6 +982,13 @@ The `method_field` function generates an HTML `hidden` input field containing th
         {{ method_field('DELETE') }}
     </form>
 
+<a name="method-now"></a>
+#### `now()` {#collection-method}
+
+The `now` function create a new Carbon instance for the current time:
```
Comment (browner12) [inline helpers.md]: `create` --> `creates`

File: helpers.md [hunk 2]
```diff
@@ -1067,6 +1076,13 @@ If no Closure is passed to the `tap` function, you may call any method on the gi
         'email' => $email
     ]);
 
+<a name="method-today"></a>
+#### `today()` {#collection-method}
+
+The `today` function create a new Carbon instance for the current date:
```
Comment (browner12) [inline helpers.md]: `create` --> `creates`

Comment (nyanlynnhtut) [reply helpers.md]: Thanks @browner12 bro

**Pull request closed without merging**